app/services/answer_star_service.py

```
@@ -0,0 +1,62 @@
+            user_answer=None,
+            has_answer=False,
+        )
+        db.add(new_answer)
+
+    try:
+        await db.commit()
+    except SQLAlchemyError:
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="별표 상태 업데이트 중 데이터베이스 오류가 발생했습니다.",
+        )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)